Client/Frontend/Widgets/ChevronView.swift

```diff
@@ -85,25 +85,29 @@ class ChevronView: UIView {
     }
 
     fileprivate func drawUpChevronAt(_ origin: CGPoint, strokeLength: CGFloat) -> UIBezierPath {
-        return drawChevron(CGPoint(x: origin.x-strokeLength, y: origin.y+strokeLength),
+        return drawChevron(
+            CGPoint(x: origin.x-strokeLength, y: origin.y+strokeLength),
```

Comment:
Should there be spaces before and after -/+?

Reply:
good point!